protos/sources/vega/governance.proto

```diff
@@ -238,6 +238,8 @@ message UpdateSpotMarketConfiguration {
 message UpdateInstrumentConfiguration {
   // Instrument code, human-readable shortcode used to describe the instrument.
   string code = 1;
+  // Instrument name
+  string name = 2;
```

Comment:
Isn't this used as well for the API? in which case it needs to be set optional to be comaptible to previous protos where this fields didn't exists?